HiveOperator/src/vrep/vflow/dockerfiles/examples/HiveOperator/Dockerfile

```diff
@@ -1,12 +1,23 @@
-FROM $com.sap.python27
+FROM python:3.6.4-slim-stretch
```

Comment:
What is the reason to not derive from the pre-delivered base image $com.sap.sles.base, see https://help.sap.com/viewer/aff95eebc2e04c44816e6ff0d21c3c88/3.0.latest/en-US/d49a07c5d66c413ab14731adcfc4f6dd.html

Reply:
Because base image does not have apt-get